### PR TITLE
fix #37419 rbc.ua

### LINKED ISSUE #37419: rbc.ua
### Issue URL (Ads)
[https://www.rbc.ua/rus/tag/vkontakte-24052015](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.rbc.ua%2Frus%2Ftag%2Fvkontakte-24052015)
### Comment
Username: @spok1
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Ext
AdGuard version: | 3.2.1
Browser: | Chrome
Stealth mode options: | Hide your search queries, Send Do-Not-Track header, Remove X-Client-Data header from HTTP requests, Strip URLs from tracking parameters, Hide your Referrer from third-parties
Filters: | AdGuard Russian,<br>AdGuard Annoyances,<br>AdGuard Base,<br>AdGuard Tracking Protection,<br>AdGuard Social Media

### PR DESCRIPTION
**Pull request by tvinzz**
https://github.com/AdguardTeam/AdguardFilters/issues/37419

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1946,6 +1946,7 @@ windowstechblog.nl#$#.onp-sl-blur-area[style="filter: blur(5px);"] { filter: non
 gigadownloads.co,techsini.com##.onp-sl-overlap-box
 /pandalocker.$script,domain=lanka.ru
 jokerbet27.tv###wh-widget-send-button
+rbc.ua##a.telegram-link[target="_blank"]
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]: `[target="_blank"]` is redundant here

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (tvinzz) [reply AnnoyancesFilter/sections/annoyances.txt]: deleted redundant part

File: AnnoyancesFilter/sections/annoyances.txt [hunk 2]
```diff
@@ -1946,6 +1946,7 @@ windowstechblog.nl#$#.onp-sl-blur-area[style="filter: blur(5px);"] { filter: non
 gigadownloads.co,techsini.com##.onp-sl-overlap-box
 /pandalocker.$script,domain=lanka.ru
 jokerbet27.tv###wh-widget-send-button
+rbc.ua##a.telegram-link
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt]: tag is not necessary here

Comment (tvinzz) [reply AnnoyancesFilter/sections/annoyances.txt]: @Alex-302,
removed tag.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**